properties.schema

```diff
@@ -6,24 +6,9 @@
   "globals": {
     "openButtonText": {
```

Comment:
`openButtonText` should be nested within `_extensions`

Reply:
Yes should, the schema applies from the root of the course json.